### ISSUE TITLE
Missing alt text on images; required for accessibility

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov-pra#184]
On the [PRA approval process](https://federalist-proxy.app.cloud.gov/preview/gsa/digitalgov-pra/bh-digital-banner/clearance-process/ ) page: 

The eight gold check mark images in the table (?) need _alt text_ in the code to be accessible for blind and low-vision users so that they know what they are, and where they are.

Suggestions:

Step 1 has one check — 
**alt text**: For Step 1, this gold check mark is in the first column, under Agency.

Step 2 has two checks — 
**alt text**: For Step 2, this gold check mark is in the first column, under Agency.
**alt text**: For Step 2, this gold check mark is in the second column, under Public.

Step 3 has one check — 
**alt text**: For Step 3, this gold check mark is in the first column, under Agency.

Step 4 has three checks — 
**alt text**: For Step 4, this gold check mark is in the first column, under Agency.
**alt text**: For Step 4, this gold check mark is in the second column, under Public.
**alt text**: For Step 4, this gold check mark is in the third column, under OMB.

Step 5 has one check — 
**alt text**: For Step 5, this gold check mark is in the third column, under OMB.

**Comment by amandacostello** [comment]
Not sure how to add this alt text for the images; reassigning to @cmajel
